code/builders/builder-vite/src/codegen-importfn-script.ts

```diff
@@ -51,5 +51,5 @@ export async function generateImportFnScriptCode(options: Options): Promise<stri
 
   // We can then call toImportFn to create a function that can be used to load each story dynamically.
   // eslint-disable-next-line @typescript-eslint/return-await
-  return await toImportFn(options.projectRoot || process.cwd(), stories);
+  return await toImportFn(stories);
```

Comment:
If you keep the `root` argument in `toImportFn` then
```suggestion
  return await toImportFn(process.cwd(), stories);
```
should work, or not?

Simplifies the tests a bit as you don't have to setup a mock on `cwd`.

Reply:
That's a good point actually, but I think externalising `root` from the function can inadvertently communicate to the _next maintainer_ that it _is_ possible to have another root than CWD, which is currently not true, and I don't think that is the right tradeoff. I think the tests are _okay_, not horrible at the moment.